docs/wiki/feature/ai.md

```diff
@@ -14,21 +14,16 @@ version:
 
 ## 1. Overview
 
-### 1.1 Adjusted AI skill values
-The idea here is to reduce the AI's godlike aiming capabilities while retaining its high intelligence. The AI should be smart enough to move through a town, but also be 'human' in their reaction time and aim.
```

Comment:
On the feature page, we should probably mention that this feature was dropped in version 3.10.